elasticapm/transport/http.py

```diff
@@ -51,12 +51,12 @@ def __init__(self, url, **kwargs):
         super(Transport, self).__init__(url, **kwargs)
         url_parts = compat.urlparse.urlparse(url)
         pool_kwargs = {"cert_reqs": "CERT_REQUIRED", "ca_certs": certifi.where(), "block": True}
-        if self._server_cert:
+        if self._server_cert and not url.startswith("http://"):
```

Comment:
You could use `url_parts.scheme` here and on line 59 since we already have the url nicely parsed :)

Reply:
Done.